packages/apputils/src/commandpalette.ts

```diff
@@ -63,6 +65,9 @@ export class ModalCommandPalette extends Panel {
     this.hide();
     this._commandPalette.inputNode.value = '';
     this._commandPalette.refresh();
+    if (this._options.restore) {
+      this._options.restore();
+    }
```

Comment:
I think this could be one line?
```suggestion
    this._options.restore?.();
```